### DISCUSSION TITLE
Light temperature selection in kelvin

### DISCUSSION THREAD

**Discussion by timeames85** [Blackymas/NSPanel_HA_Blueprint#1524] [Ideas]
With this now more supported in the HA UI (https://github.com/home-assistant/frontend/pull/18627), how tricky might it be to allow the colour temp slider to use in Kelvin (e.g. 2700-6500k) rather than mired (153-500)? Maybe a configurable option?

I'd not used the mired unit before HA and my brain still seems to default to K when thinking about lighting, maybe due to bulb packaging and having a Philips Hue setup.

**Comment by edwardtfn** [comment]
- #1525
